lib/_debugger.js

```diff
@@ -65,15 +65,13 @@ function Protocol() {
 }
 exports.Protocol = Protocol;
 
-
```

Comment:
I don't think the current linter enforces this, but if possible, I'd loosen that rule to permit double empty lines in `jscs`.